core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CommonSettingsAndTasksPlugin.scala

```diff
@@ -42,11 +42,15 @@ object CommonSettingsAndTasksPlugin extends AutoPlugin {
   // common definitions
   final val CloudflowLocalConfigFile = ".lightbend/cloudflow/pipectl.json"
   final val CloudflowDockerBaseImage = "lightbend/cloudflow-base:1.3.0-M1-spark-2.4.4-flink-1.9.1-scala-2.12"
+  // used for internal release
+  final val CloudflowBintrayReleasesRepoUrl = "https://lightbend.bintray.com/cloudflow"
```

Comment:
something like `CloudflowBintrayTestReleasesRepoUrl` could be more precise:

Reply:
Is this going to be shipped with the plugin? I think this should be only enabled for internal use somehow.

Reply:
@skonto Strictly speaking it's not for test, it's for an internal release which, once done, all team members can use - hence the name.

Regarding shipping, why not ? The repository where we publish is a public one - hence people can use it from outside lightbend as well. And besides, how can we keep it only for internal use ?